.github/workflows/kit.yml

```diff
@@ -55,6 +55,7 @@ jobs:
         with:
           version: ${{ env.GOLANGCI_LINT_VER }}
           skip-cache: true
+          args: --build-tags=unit
```

Comment:
Please modify the `.golangci-lint.yaml` file instead, so it runs when developers run linter locally too

Reply:
Yes, good shout! Done